### PR TITLE
Modified as per discussion with @niesink

### PR DESCRIPTION
**Pull request by cliftonc**
https://github.com/stichtingsem/technology-prototype/issues/8#issuecomment-675294177

- Also modified sisdata to keep it consistent.
- @niesink

### CODE REVIEW

**Comment by cliftonc** [comment]
@niesink it would be great if you can skim over this today?  I am hoping to get these merged before tomorrow.

File: reference/sisdata.v1.yaml [hunk 1]
```diff
@@ -245,23 +253,26 @@ components:
...
           $ref: '#/components/schemas/student'
-        class:
-          $ref: '#/components/schemas/class'
+        group:
+          $ref: '#/components/schemas/group'
         school:
           $ref: '#/components/schemas/school'
+        schoolYear:
+          type: string
+          description: A reference to the school year for this group.
```
Comment (niesink) [inline reference/sisdata.v1.yaml:270]: I'm not sure we should add this property, as mentioned in https://github.com/stichtingsem/technology-prototype/issues/28#issuecomment-675873495

Nitpicking: If we do decide to add this property the word 'group' in the description should be changed to 'subjectChoice' I think

**Review by niesink — CHANGES_REQUESTED** [review]
Just the one thing on the schoolYear property of a subjectChoice

File: reference/sisdata.v1.yaml (hunk 1 above)
Comment (cliftonc) [reply reference/sisdata.v1.yaml]: Done

**Review by niesink — APPROVED** [review]

**Pull request merged**